### PR TITLE
Update envelope model to make container field not required

### PR DESCRIPTION
**Pull request by aliveni**

### CODE REVIEW

**Comment by nitinprabhu** [comment]
Good point If it is null it should work the way it works now i.e use Jurisdiction

**Comment by doncem** [comment]
There's annotation at the beginning of class:

```java
@JsonIgnoreProperties(ignoreUnknown = true)
```

in case property is not present - it'll be automatically null

ah, but all the `container` usage must be reviewed. which is exception record case type i believe. revoking my instant approval

File: src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/model/Envelope.java [hunk 1]
```diff
@@ -26,7 +26,7 @@ public Envelope(
         @JsonProperty(value = "case_ref", required = true) String caseRef,
         @JsonProperty(value = "po_box", required = true) String poBox,
         @JsonProperty(value = "jurisdiction", required = true) String jurisdiction,
-        @JsonProperty(value = "container", required = true) String container,
+        @JsonProperty(value = "container") String container,
```
[the remark this answers is not in the document]
Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/model/Envelope.java]: imho
`@JsonProperty(value = "container", required = false)`
better than
`@JsonProperty("container")`
and
`@JsonProperty(value = "container")`
given what other params look like

**Comment by aliveni** [comment]
This pr is no longer needed. All code related to ER case type are merged and no issues reported.

**Pull request closed without merging**